app/models/comfy/cms/page.rb

```diff
@@ -1,6 +1,9 @@
 require ComfortableMexicanSofa::Engine.root.join('app', 'models', 'comfy', 'cms', 'page.rb')
```

Comment:
If you remove the rb the require will not work. I tested here on 2.1.2. and returns false and the page from comfy was not required.